CryptoPrices/Sources/Home/Package.swift

```diff
@@ -16,13 +16,15 @@ let package = Package(
     dependencies: [
         .package(name: "Styleguide", path: "../Styleguide"),
         .package(url: "https://github.com/Quick/Quick", from: "6.1.0"),
-        .package(url: "https://github.com/Quick/Nimble", from: "11.2.1")
+        .package(url: "https://github.com/Quick/Nimble", from: "11.2.1"),
+        .package(name: "MyCoin", path: "../MyCoin")
     ],
     targets: [
         .target(
             name: "Home",
             dependencies: [
-                .product(name: "Styleguide", package: "Styleguide")
+                .product(name: "Styleguide", package: "Styleguide"),
+                .product(name: "MyCoin", package: "MyCoin")
```

Comment:
The `Home` and `MyCoin` are modules that represent their own features, they belong to the same _horizontal layer_. They shouldn't depend on each other.
Maybe the name `MyCoin` is a little bit confusing and misleading here 🙇  but it was created for the _detail_ screen.
Kindly put all `Home`'s related codes (`MyCoinView`, `MyCoinItemView`) into the `Home` package 🙏

Reply:
🙈 my bad, I thought MyCoin is a module for MyCoins. Fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/32/commits/5cf90f65644ed6288a7ff72cf0edd66e9b060dc3. 🙏